src/style.css

```diff
@@ -0,0 +1,109 @@
+/* #region html */
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-display: swap;
+  font-style: normal;
```

Comment:
Default value, you can just remove it
```suggestion
```

Reply:
Thank you!)
I will do so in the following tasks